installers/macOS/setup.py

```diff
@@ -100,24 +100,31 @@ def make_app_bundle(dist_dir, make_lite=False):
     pyls :
         <path>/Contents/MacOS/python: No module named pyls
         Note: still occurs in alias mode
+    pyls_black :
+        Mandatory: pyls_black >=0.4.6 : None (NOK)
+    pyls_spyder :
+        Mandatory: pyls_spyder >=0.1.1 : None (NOK)
     qtawesome :
         NotADirectoryError: [Errno 20] Not a directory: '<path>/Resourses/lib/
         python38.zip/qtawesome/fonts/fontawesome4.7-webfont.ttf'
+    setuptools :
+        Mandatory: setuptools >=39.0.0 : None (NOK)
+    sphinx :
+        No module named 'sphinx.builders.changes'
     spyder :
         NotADirectoryError: [Errno 20] Not a directory: '<path>/Resources/lib/
         python38.zip/spyder/app/mac_stylesheet.qss'
     spyder_kernels :
         No module named spyder_kernels.console.__main__
-    sphinx :
-        No module named 'sphinx.builders.changes'
 
    """
     build_type = 'lite' if make_lite else 'full'
     logger.info('Creating %s app bundle...', build_type)
 
     PACKAGES = ['alabaster', 'astroid', 'ipykernel', 'IPython', 'jedi',
-                'jinja2', 'keyring', 'parso', 'pygments', 'pyls', 'qtawesome',
-                'spyder', 'spyder_kernels', 'sphinx']
+                'jinja2', 'keyring', 'parso', 'pygments', 'pyls', 'pyls_black',
+                'pyls_spyder', 'qtawesome', 'setuptools', 'sphinx', 'spyder',
+                'spyder_kernels']
```

Comment:
You're missing `three-merge` here, which is also a new dependency for 4.2

Reply:
I confirmed that `three-merge` is discovered by py2app and is included in the macOS app, so no further action is required.
<img width="823" alt="Screen Shot 2020-11-03 at 1 47 14 PM" src="https://user-images.githubusercontent.com/9618975/98044042-1ad2d480-1ddb-11eb-8813-7472782f1bc6.png">

Reply:
Ok, thanks!